### ISSUE TITLE
[LINT_BUG]: Not linting on non-existing package

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#107]
### box.linters version

0.9.1

### Sample source code to lint

```R
box::use(sttttr, )
```

### Lint command used

```R
lintr::lint(
  code,
  linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters)
)
```

### Lint result

```
<text>:1:10: warning: [box_unused_att_pkg_linter] Attached package unused.
box::use(stttr, )
         ^~~~~
```

### Expected result

Should lint on non-existing package
